src/coreclr/jit/lsraarm64.cpp

```diff
@@ -1091,55 +1091,52 @@ int LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree)
             }
         }
     }
-    else
+    else if (intrin.op2 != nullptr)
```

Comment:
Since you are handling *"contained Vector.Zero"* in `LinearScan::BuildOperandUses` you don't need this part of the change, right?

Reply:
This is easier to see if you hide whitespace changes. It's actually a zero diff and is just being changed from:
```csharp
else
{
    if (intrin.op2 != nullptr)
    {
        // ...
    }
}
```

to:
```csharp
else     if (intrin.op2 != nullptr)
{
    // ...
}
```